components/nimbus/src/lib.rs

```diff
@@ -796,6 +797,72 @@ impl NimbusTargetingHelper {
 
 type JsonObject = Map<String, Value>;
 
+pub struct IntervalData {
```

Comment:
Let's put this in it's own module and not in the main lib.rs file. Then we can import the module where we need it.